python/cudf/cudf/utils/ioutils.py

```diff
@@ -473,8 +473,36 @@
 engine : {{ 'auto', 'cudf', 'cudf_experimental', 'pandas' }}, default 'auto'
     Parser engine to use. If 'auto' is passed, the engine will be
     automatically selected based on the other parameters.
-orient : string,
-    Indication of expected JSON string format (pandas engine only).
+lines : boolean, default False
+    Read the file as a json object per line.
+dtype : boolean or dict, default True
+    If True, infer dtypes, if a dict of column to dtype, then use those,
+    if False, then don't infer dtypes at all, applies only to the data.
```

Comment:
```suggestion
    If True, infer dtypes for all columns; if False, don't infer dtypes at all; if a dict,
    provide a mapping from column names to their respective dtype (any missing
    columns will have their dtype inferred).
    
    
    Applies only to the data.
```

WDYT? 

I am not sure what the "applies only to the data" comment is meant to mean so I left it as a hanging sentence at the end.